src/Sloth.Api/appsettings.Development.json

```diff
@@ -8,6 +8,6 @@
     }
   },
   "ConnectionStrings": {
-    "SlothDatabase": "Data Source=EPUAKYIW1342\\SQLEXPRESS;Initial Catalog=Sloth;User Id=sa;Password=Password1;"
+    "SlothDatabase": "Data Source=localhost;Initial Catalog=Sloth;User Id=Sloth;Password=Sloth;"
```

Comment:
please, it's your local setup, don't commit it

Reply:
sorry